### ISSUE TITLE
Silence disambiguation symbol not being added

### ISSUE THREAD

**Issue by danpovey** [kaldi-asr/kaldi#904]
I just noticed in prepare_lang.sh line 378 that it says:
```
  utils/make_lexicon_fst.pl --pron-probs $tmpdir/lexiconp.txt $sil_prob $silphone | \
    fstcompile --isymbols=$dir/phones.txt --osymbols=$dir/words.txt \
```
so it is not passing in the silence disambiguation symbol `'#'$ndisambig`.
This could lead to determinization failures in setups that have a word pronounced as silence.
It should probably read:
```
  utils/make_lexicon_fst.pl --pron-probs $tmpdir/lexiconp.txt $sil_prob $silphone '#'$ndisambig | \
    fstcompile --isymbols=$dir/phones.txt --osymbols=$dir/words.txt \
```
Someone needs to test this change in some setup and see if there is any effect on graph size.
It should be safe to fix but it's worth testing first.

**Comment by danpovey** [comment]
... I've asked Hang Lv to do this.

**Comment by danpovey** [comment]
Addressed in PR #934
